clusterloader2/pkg/prometheus/manifests/prometheus-prometheus.yaml

```diff
@@ -34,6 +34,10 @@ spec:
     fsGroup: 2000
     runAsNonRoot: true
     runAsUser: 1000
+  tolerations:
+  - key: "monitoring"
+    operator: "Exists"
+    effect: "PreferNoSchedule"
```

Comment:
Prefer won't force anything - is that really what you want?
Shouldn't we use `NoSchedule`

Reply:
Yes, I don't think it's going to be a problem if we schedule one or two hollow nodes or other infrastructure pods on this node. I don't have a strong preference on this though.

Reply:
Done